tv/.env

```diff
@@ -1,3 +1,6 @@
-# CDP
+# CMP
 NEXT_PUBLIC_CMP_PREVIEW_ENDPOINT_URL=
-NEXT_PUBLIC_CMP_PREVIEW_API_KEY=
\ No newline at end of file
+NEXT_PUBLIC_CMP_PREVIEW_API_KEY=
+
+# Website
+NEXT_PUBLIC_WEBSITE_URL=
```

Comment:
This variable must be added to Vercel deployments in `docker\build\init\Jobs\DeployToVercel.cs` similar to how it is done for the kiosk.